Gemfile.lock

```diff
@@ -566,9 +570,9 @@ GEM
       method_source
       rake (>= 12.2)
       thor (~> 1.0)
-    rainbow (3.0.0)
+    rainbow (3.1.1)
```

Comment:
### [rainbow](https://github.com/sickill/rainbow)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.0.0 → 3.1.1

[_~~change-log~~, [source-code](http://github.com/sickill/rainbow), [gem-diff](https://my.diffend.io/gems/rainbow/3.0.0/3.1.1)_]





<details>
<summary>Commits</summary>

A change of **29** commits. See the full changes on [the compare page](https://github.com/ku1ik/rainbow/compare/v3.0.0...v3.1.1).

These are the first 10 commits:
- (45128cf) [Update changelog](https://github.com/ku1ik/rainbow/commit/45128cf9f64d7eaea82b464efe5ff7cd24ad50c8)
- (b78e39d) [[CI] Test against new Ruby 2.5 and patched](https://github.com/ku1ik/rainbow/commit/b78e39de1947dac40220688523ea06922f9102f4)
- (21b18bc) [Merge pull request #81 from nicolasleger/patch-1](https://github.com/ku1ik/rainbow/commit/21b18bc5ecc9bad6febe035a2f78738cdaf70716)
- (e4f74cf) [Add unit test for non hexadecimal string](https://github.com/ku1ik/rainbow/commit/e4f74cfb61c2e394be44554141f47da321bd6298)
- (0880e17) [Add hexadecimal format checking](https://github.com/ku1ik/rainbow/commit/0880e173448132f6e28a967a17ec8150aef179a3)
- (c7cb8da) [Make rubocop happy](https://github.com/ku1ik/rainbow/commit/c7cb8dae63639693c6aab87254677249f00d98cb)
- (4bba7e3) [Merge pull request #83 from chiting/add-hex-check](https://github.com/ku1ik/rainbow/commit/4bba7e3798394abb2647082d414e7d267790bdd3)
- (de9d94b) [Simplify `wrap_with_sgr` method (#85)](https://github.com/ku1ik/rainbow/commit/de9d94be5ac3d9798b97b2caf700afd72c7bda7f)
- (6680596) [introducing #cross_out (#84)](https://github.com/ku1ik/rainbow/commit/6680596e25e75fff9311423e6c06d7bcdbff7e06)
- (62a0a3e) [Add test with non-existent :color (#86)](https://github.com/ku1ik/rainbow/commit/62a0a3ed0ed79011515674e7912ccf39ba6a1569)


</details>